packages/frontend-shared/src/styles/shared.scss

```diff
@@ -10,17 +10,6 @@
     local("Roboto-Light"), local("DroidSans"), local("Tahoma");
 }
 
-/* Set up Roboto for modern browsers, all weights */
```

Comment:
This was not doing anything anyway, and it looks like something changed internally in Vite that was causing this to 404, so I just deleted it.